CHANGELOG

```diff
@@ -1,3 +1,10 @@
+Version 0.3.1 (2019-09-19)
+--------------------------
+Update scala to 2.11.1 and sbt to 0.13.18
+Update referers.yml to latest version from https://s3-eu-west-1.amazonaws.com/snowplow-hosted-assets/third-party/referer-parser/referers-latest.yml
```

Comment:
Does the new yml come from there? Or is it meant to go there?

Reply:
It comes from there.